docs/advanced-configuration.md

```diff
@@ -157,6 +158,37 @@ the following line to your custom values.yaml:
 priorityClassName: splunk-otel-agent-priority
 ```
 
+## EKS Fargate support
+
+If you want to run the Splunk OpenTelemetry Collector in [Amazon Elastic Kubernetes Service
+with Fargate profiles](https://docs.aws.amazon.com/eks/latest/userguide/fargate.html),
+make sure to set the required `distribution` value to `eks/fargate`:
+
+```yaml
+distribution: eks/fargate
+```
+
+**NOTE:** Fluentd and Native OTel logs collection are not yet automatically configured in EKS with Fargate profiles
+
+This distribution will operate similarly to the `eks` distribution but with the following distinctions:
+
+1. The Collector agent daemonset is not applied since Fargate doesn't support daemonsets. Any desired Collector instances
+running as agents must be configured manually as sidecar containers in your custom deployments. This includes any application
+logging services like Fluentd. We recommend setting the `gateway.enabled` to `true` and configuring your instrumented
+applications to report metrics, traces, and logs to the gateway's `<installed-chart-name>-splunk-otel-collector` service address if no
+agent instances are used in your cluster. Any desired agent instances that would run as a daemonset should instead run as sidecar containers in your pods.
+2. The Collector's ClusterRole for `eks/fargate` will allow the `patch` verb on `nodes` resources for the default API groups. This is to allow
```

Comment:
The first item includes a recommendation to users with some specific needs, but the next two are more like implementation details of some peculiarities without mentioning a reason for having them.

Maybe we can group the next two items into one and let user now why this is needed pointing to the EKS/Fargate limitation? I'm thinking of a list of two items, each of them describes a limitation and appropriate customization applied to the chart to resolve it. WDYT?

Reply:
The intention is to describe the objects and the reason for their existing considering that this is a bit unintuitive compared to the deployment. If it's not clear then I can try to make this more readable/information. How's this?

Reply:
Thanks. It's much cleaner now. Added just one comment